packages/styles/scss/components/horizontalrule/_horizontalrule.scss

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '~@carbon/themes/scss/themes';
+@import '~carbon-components/scss/globals/scss/vars';
+@import '~carbon-components/scss/globals/scss/helper-mixins';
+@import '~carbon-components/scss/globals/scss/layout';
+@import '~carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/import-once/import-once';
+@import '~carbon-components/scss/globals/scss/css--reset';
+
+$low-contrast: $ui-03;
+$medium-contrast: $ui-04;
+$high-contrast: $ui-05;
+
+@mixin horizontalrule {
+    .#{$prefix}--hr {
+        margin: $spacing-05 0;
```

Comment:
I thought I added this comment in the last pass of review, but these tokens are actually legacy at the moment. Carbon recently went through and namespaced all of their tokens to prevent conflicts. `$carbon--spacing-0x`. It might be worth updating and trying to be vigilant about it. There area couple more like this in this file too.